packages/uxpin-merge-cli/src/steps/serialization/component/implementation/typescript/component/isArrowFunction.ts

```diff
@@ -0,0 +1,9 @@
+import * as ts from 'typescript';
+import { ComponentDeclaration } from './getPropsTypeAndDefaultProps';
+
+export function isArrowFunction(node:ComponentDeclaration | ts.VariableStatement):boolean {
+  return (
+    ts.isVariableStatement(node) &&
+    ts.isArrowFunction((node as ts.VariableStatement).declarationList.declarations[0].initializer as ts.ArrowFunction)
```

Comment:
Maybe safer will be
```
node.declarationList &&
node.declarationList.declarations && 
node.declarationList.declarations[0] &&
ts.isArrowFunction((node as ts.VariableStatement).declarationList.declarations[0].initializer as ts.ArrowFunction)
```
unless you are sure about this object :)

Reply:
True! I've made it safer :)